### ISSUE TITLE
Signature test failures

### ISSUE THREAD

**Issue by imnotbrandon** [punk-security/dnsReaper#192]
Signature tests often have transient issues (e.g captcha challenge pages) and fail on unrelated changes. Need a better workflow to deal with them, as pull requests shouldn't fail just because an unrelated signature has an issue.

**Comment by SimonGurney** [comment]
I've fixed 1 sig (update by provider) and removed 1 sig (no longer valid for wix).

The Frontify one applies bot detection to Github IP space... but it works local. Do we filter it out from the testing specifically?

#196

**Comment by imnotbrandon** [comment]
Yeah I think we need to have an exclude list in CI and rely on manual testing unfortunately. Even if we moved to a self hosted runner on aws or something we'd likely face the same issue
